### ISSUE TITLE
Explore babeldown once more mature

### ISSUE THREAD

**Issue by maelle** [rladies/rladies.github.io#135]
https://docs.ropensci.org/babeldown/

**Comment by drmowinckels** [comment]
Babeldown is maturing and now has  guide on how to set up!

https://ropensci.org/blog/2023/09/26/how-to-translate-a-hugo-blog-post-with-babeldown/

We should start working on this asap, as the translation team will be coming in soon, and providing some backbone for them would be good!

As part of this, if we get it working, we should also change the "translation" placeholder from "sorry for not being translated" to "this page is auto-translated with DeepL"

**Comment by maelle** [comment]
Happy to help, of course!

**Comment by yabellini** [comment]
Me too :-)

**Comment by maelle** [comment]
yep, happy to participate in a meeting to kick things off, and to help more afterwards. Did R-Ladies get a DeepL account?

**Comment by maelle** [comment]
deepl is what babeldown uses :-)

**Comment by shannonpileggi** [comment]
got it, thank you!! very cool!

**Comment by drmowinckels** [comment]
We have a deeply account, login details in global 1pass ☺️
